threads-csharp/IntroductionToSemaphoreInCsharp/IntroductionToSemaphoreInCsharp/ExampleWithSemaphore.cs

```diff
@@ -0,0 +1,36 @@
+﻿using IntroductionToSemaphoreInCsharp.Models;
+using System.Globalization;
+
+namespace IntroductionToSemaphoreInCsharp;
+
+public class ExampleWithSemaphore
+{
+    private static List<string> _sharedResource = new List<string>();
+    private static readonly Semaphore _semaphore = new Semaphore(initialCount: 3, maximumCount: 3);
```

Comment:
Please utilize the new() and new(initialCount: 3, maximumCount: 3)
We can add readonly modifier to the list.

Reply:
Please check the code for other occurences of this as well.

Reply:
Done. changed as below:
```
private static readonly List<string> _sharedResource = [];
private static readonly Semaphore _semaphore = new(initialCount: 3, maximumCount: 3);
```
```
private static readonly List<string> _sharedResource = [];
private static readonly SemaphoreSlim _semaphoreSlim = new(3, 3);
```